### DISCUSSION TITLE
Nvidia Shield Controller

### DISCUSSION THREAD

**Discussion by Doonut** [cathery/sys-con#32] [Controller requests]
Mines a 2017, but the 2015 guys may want some love too

its
```Bus 001 Device 005: ID 0955:7214 NVIDIA Corp. NVIDIA Controller v01.04```
under lsusb

Im willing to contribute, but if your going to add Xinput/Dinput support, maybe we shouldnt be just adding tons of Vendor/Product IDs

**Comment by cathery** [comment]
Try if it works on this build https://files.catbox.moe/nefssl.rar

**Comment by Doonut** [comment]
nope, after looking some more, it looks like even on windows it requires GeForce Experience installed so it may be a stretch

**Comment by trbocode** [comment]
It works on the Wii U tools "HID to VPAD" and "Nintendont" if that's of any help.

**Comment by cathery** [comment]
Does it work on the latest release 0.4.2?

**Comment by trbocode** [comment]
Nope. XBOX 360 Wireless adapter works fine, so I'm certainly running the software. The controller rumbles, but doesn't show up on switch. 2017 ver.

**Comment by cathery** [comment]
Rip. Must need additional drivers then

**Comment by trbocode** [comment]
As I said - it works on the Wii U tools, through. Which most certainly don't use any drivers.

**Comment by leoaraucosta** [comment]
Any news about support nvidia shield controller? Thanks.
